packages/core/helpers/DataTransform.ts

```diff
@@ -11,7 +14,7 @@ export class DataTransform {
 
   //Performs standardizations that can be completed automatically without use input
   autoStandardize(data) {
-    const errorsFound = []
+    const errorsFound: any[] = []
 
     // Empty data
     if (0 === data.length) {
```

Comment:
I'm receiving an error at this point where data doesn't have a length. Consider using a short circuit ternary or checking for data before this point.

Reply:
I tried using the default example ` <div class="react-container" data-config="/examples/default.json"></div>` I'm not seeing any error here though.